Neumannova_Pavla_PS1.R

```diff
@@ -0,0 +1,47 @@
+# Problem 1
+# print actual time
+
+Sys.time()
+
+# create a list �bad list� with 3 positions
+
+badlist <- list(na = c(NA, NA, NA), chars = c("a", "b", "c"), bools = c(FALSE, FALSE, TRUE))
+badlist
+
+# create a data frame
+
+happyframe <- data.frame(na = c(NA, NA, NA), chars = c("a", "b", "c"), bools = c(FALSE, FALSE, TRUE))
+happyframe
+
+# Problem 2
+# lenght of numeric vector input
+x <- c(1, 2, 3, 4)
+length(x)
+
+# not using len function
+vector <- c(1,2,3,4,5,6,7,8,9)
+diff(range(vector))+1
+
+# boolean vector
+ifelse(vector %% 3 == 0, "TRUE", "FALSE")
+
+# fruits
+ifelse(vector > 5, "RED", "NON-RED")
```

Comment:
Again, your code here returns a value "RED" is element is bigger than 5.
However, you were supposed to have checked whether the fruit is 
1)both red and bigger than 5 -> assign TRUE
2) both non-red and smaller than 5 -> assign TRUE
3) otherwise -> FALSE
So output should be boolean vector, i.e. containing only TRUE and FALSE